api/src/repositories/draft-repository.ts

```diff
@@ -0,0 +1,121 @@
+import { QueryResult } from 'pg';
+import SQL, { SQLStatement } from 'sql-template-strings';
+import { z } from 'zod';
+import { ApiExecuteSQLError } from '../errors/api-error';
+import { BaseRepository } from './base-repository';
+
+export const WebformDraftList = z.object({
```

Comment:
Maybe `WebformDraft` would be a more apt name, since it's not actually a list?

Reply:
yeah, good point